po/uk_UA.po

```diff
@@ -10328,27 +10328,35 @@ msgid "Group %u selected - %u Unit"
 msgid_plural "Group %u selected - %u Units"
 msgstr[0] "Групу %u обрано - %u Підрозділ"
 msgstr[1] "Групу %u обрано - %u Підрозділів"
+msgstr[2] "Групу %u обрано - %u Підрозділів"
```

Comment:
Any new translations you have not checked should be marked as fuzzy.

Reply:
It doesn't appear to be possible to mark a single form of a plural form string as fuzzy. For these plural form updates, they should (at worst) be the same output as the prior iteration of the .po files. (But now with the ability for someone to alter the additional plural forms, if needed.)